polly/lib/CodeGen/CodeGeneration.cpp

```diff
@@ -233,6 +238,30 @@ static bool generateCode(Scop &S, IslAstInfo &AI, LoopInfo &LI,
   NodeBuilder.allocateNewArrays(StartExitBlocks);
   Annotator.buildAliasScopes(S);
 
+  // The code below annotates the "llvm.loop.vectorize.enable" to false
+  // for the code flow taken when RTCs fail. Because we don't want the
+  // Loop Vectorizer to come in later and vectorize the original fall back
+  // loop when 'polly-annotate-metadata-vectorize' is passed.
+  if (PollyVectorizeMetadata) {
```

Comment:
```suggestion
```
I'd be fine if the loop vectorizer is always disabled for fallback code. Would it means too many test updates?

Reply:
I am seeing around 19 failures.
Should we have it as separate patch.
Or can it be part of this patch itself ?

Reply:
@Meinersbur Please provide inputs on if we have to add the test case changes for 19 failures as separate patch.